controllers/app.controllers.js

```diff
@@ -1,8 +1,20 @@
-const { selectTopics } = require('../models/app.models')
+const { selectTopics, selectArticle } = require('../models/app.models')
 
 exports.getTopics = (req, res, next) => {
     selectTopics()
     .then((result) => {
         res.status(200).send({ topics: result })
     })
 }
+
+exports.getArticleById = (req, res, next) => {
+    const { article_id } = req.params
+    selectArticle(article_id)
+    .then((article) => {
+        if (!article) {
+            return Promise.reject({ status: 404, msg: 'Article does not exist' })
```

Comment:
This promise.reject should really be in our model. The controller ideally will just invoke the model, then send the response. 👍